src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -334,22 +334,30 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Catalog updates/changes
 
-(defn catalog-metadata
+(defn latest-catalog-metadata
   "Returns the id and hash of certname's catalog"
```

Comment:
Is this still what it returns? Looks like certname_id and producer_timestamp have been added.